ADAL/tests/integration/AADAuthorityValidationIntegrationTests.m

```diff
@@ -777,5 +754,29 @@ - (void)testAcquireTokenSilent_whenDifferentPreferredCacheAndTokenFails_shouldRe
     XCTAssertNil([tokenCache getAT:authority]);
 }
 
+#pragma mark - Private
+
+- (ADTokenCache *)createMRRTTokenCache:(NSString *)preferredAuthority
```

Comment:
I'm confused here :) What is this "doNotUseAuthority" used for?

Reply:
dunno for sure, I just replaced static function with private method, see here:https://github.com/AzureAD/azure-activedirectory-library-for-objc/pull/1161/files#diff-757429c843582dea38edbe3bda763c7bL598

As far as I understand, 'doNotUseAuthority' is used for another item in cache and in tests we check that this item was not updated when preferredAuthority is updated. I agree that name is not self-explaining, but in order to keep this pr small, let revise those tests later. Please let me know if you don't agree.

Reply:
I agree :) I thought it's new code that you added.